EN/3_practice.md

```diff
@@ -28,7 +28,7 @@ Steps:
 4. Write some sample code to try your new method in `ruby/test.rb`, then build and run with `make run`.
 5. Add a test in `ruby/test/ruby/test_array.rb`. These tests are written in the minitest format.
 6. `$ make test-all` will run the test code you wrote. However, it runs a tremendous number of ruby tests, so you may want to run only the Array-related tests.
-  * `$ make test-all TESTS='ruby/test_array.rb'` will test only `ruby/test/ruby/test_array.rb`.
+  * `$ make test-all TESTS='../ruby/test/ruby/test_array.rb'` will test only `ruby/test/ruby/test_array.rb`.
```

Comment:
it should be `TESTS='ruby/test_array.rb`.

Reply:
I must have made some mistake before, both version are working for me now.